Tests/Utils/HardcodedTextProvider.swift

```diff
@@ -0,0 +1,22 @@
+// Created by Igor Katselenbogen on 08/19/22.
+// Copyright © 2022 Airbnb Inc. All rights reserved.
+
+import Lottie
+import QuartzCore
+#if os(iOS)
+import UIKit
+#endif
+
+// MARK: - HardcodedImageProvider
+
+/// An `AnimationTextProvider` that always returns a specific hardcoded text
+class HardcodedTextProvider: AnimationTextProvider {
+  // Question 4: What should we actually return here?
+  func textFor(keypathName: String, sourceText: String) -> String {
+    #if os(iOS)
+    return "text layer text"
+    #else
+    return nil
+    #endif
+  }
```

Comment:
I think this should be:

```suggestion
class HardcodedTextProvider: AnimationTextProvider {

  init(text: String) {
    self.text = text
  }
  
  func textFor(keypathName _: String, sourceText _: String) -> String {
    text
  }
  
  private let text: String

}
```

Reply:
Alrighty! [a053713](https://github.com/airbnb/lottie-ios/pull/1723/commits/a053713c2df1cbbfd7e51212074bcbab6a47afe1) ✅